socs/agents/http_camera/agent.py

```diff
@@ -1,31 +1,32 @@
 import argparse
+import json
 import os
 import shutil
 import time
 from pathlib import Path
 
 import requests
 import txaio
+import urllib3
+import yaml
+
+urllib3.disable_warnings()
```

Comment:
This helped suppress warnings due to this: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
Let me know if you know a better fix for that.

Reply:
Hmm, were you getting an `SSLError` making the HTTPS requests? If the camera insists on HTTPS I'm wondering if we can actually verify the requests. Otherwise, yeah, let's add a comment here about why we're disabling warnings with a link to that docs page.

One step further perhaps we can disable just the `InsecureRequestWarning`, as suggested [here](https://stackoverflow.com/questions/15445981/how-do-i-disable-the-security-certificate-check-in-python-requests/32282390#comment80086338_32282390).

Reply:
Yeah the full warning is here:
```
urllib3.exceptions.InsecureRequestWarning: Unverified HTTPS request is being made to host '192.168.65.xx'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
```

Reply:
Gotcha, yeah, seeing if we can disable only that warning and adding a comment would be great. The rest all looks good.